apps/address-service/package.json

```diff
@@ -32,7 +32,7 @@
     "graphql": "^16.10.0",
     "graphql-tag": "^2.12.6",
     "lodash": "^4.17.21",
-    "openid-client": "4.7.4",
+    "openid-client": "5.7.1",
```

Comment:
That's a lot of breaking change for that update. How do we check if everything is working fine?

Reply:
Overall migration is safe for us except the SBBOL integration, which will be tested on test environment